### ISSUE TITLE
Attendance Taking App Briefing 29 Nov

### ISSUE THREAD

**Issue by aahung** [cityu-hall2/hall-attendance-take-app#1]
# What's the Goals
1. Able to scan barcode on student IDs
2. Able to display student information after scanning
3. Able to communicate with a central server

# How Development Works
1. Around 5 "main programmers" will start the projects from December
2. The first working version will be out within 3 months, the working version has room to improve
3. Development is taken step by step,
4. Before each step, relevant skills or knowledge will be mentioned, and corresponding hand picked material will be provided
5. Regular report meeting (offline) will be held, in the report meeting, what has been done in each step, why and how will be illustrated by form of presentation. 

# Learning, Participation Forms
1. Observe we develop the app in live, continuously, to understand why and how to develop the app. 
3. Listen to the regular report meeting, to understand why and how to develop the app. 
2. Before a development step mentioned above, participant might feel ready, he/she can use the material provided to prepare to participate in development

# Prototyping Stage
Please volunteer to form a prototyping team, and propose a leader. Prototype can satisfy the following conditions:
1. Basic interface illustrated by lines, rectangular and other shapes, plus texts. 
2. Interactions such as button clicks can be illustrated by drawing a connection between multiple views.
3. There are many tools to prototype, pick whatever works. https://www.invisionapp.com (free), https://www.sketchapp.com (paid)
4. Prototyping won't take long, usually when you have an plan, drawing is as fast as hours. Please submit a draft before 8 Dec. 

Prototype examples: http://www.protoshare.com/features/examples/
**Prototyping team registration: https://github.com/cityu-hall2/hall-attendance-take-app/issues/2**

**Comment by WesYu** [comment]
For the goals, I think what information we are to store on the server is relevant as well. How much will the decisions made during the app development affect our website development?

**Comment by aahung** [comment]
@WesYu Good point, all 3 branches need to discuss the database scheme and reach a agreement we before relevant functions being developed.
